dist/playkit.js.map

```diff
@@ -0,0 +1 @@
AMkP,GAAN,CAAUnF,YAAV,CAAuBqQ,EAAvB,EAA2BvJ,iBAA3B;AACA7Q,YAAMkP,GAAN,CAAUnE,WAAV,CAAsB,KAAK8L,GAA3B,EAAgCuD,EAAhC;AACD;;AAED;;;;;;;;;8CAMoD;AAAA;;AAAA,UAA5BzF,OAA4B,uEAAV,EAAU;;AAClD,UAAIA,OAAJ,EAAa;AACXnX,eAAOC,IAAP,CAAYkX,OAAZ,EAAqBjX,OAArB,CAA6B,gBAAQ;AACnC;AACA,cAAMgd,SAAS,OAAK5G,cAAL,CAAoB9V,GAApB,CAAwBD,IAAxB,CAAf;AACA,cAAI2c,MAAJ,EAAY;AACVA,mBAAOC,YAAP,CAAoBhG,QAAQ5W,IAAR,CAApB;AACA,mBAAK0V,OAAL,CAAakB,OAAb,CAAqB5W,IAArB,IAA6B2c,OAAOE,SAAP,EAA7B;AACD,WAHD,MAGO;AACL;AACA,gBAAI,CAAC,OAAK5E,OAAV,EAAmB;AACjB,kBAAI;AACF,uBAAKlC,cAAL,CAAoBqC,IAApB,CAAyBpY,IAAzB,UAAqC4W,QAAQ5W,IAAR,CAArC;AACD,eAFD,CAEE,OAAOkJ,CAAP,EAAU;AACV;AACA,uBAAK8N,aAAL,CAAmB9N,CAAnB;AACD;AACD,kBAAIyT,UAAS,OAAK5G,cAAL,CAAoB9V,GAApB,CAAwBD,IAAxB,CAAb;AACA,kBAAI2c,OAAJ,EAAY;AACV,uBAAKjH,OAAL,CAAakB,OAAb,CAAqB5W,IAArB,IAA6B2c,QAAOE,SAAP,EAA7B;AACA,oBAAI,OAAOF,QAAOG,iBAAd,KAAoC,UAAxC,EAAoD;AAClD,yBAAK5G,mBAAL,CAAyB6G,GAAzB,CAA6BJ,QAAOG,iBAAP,EAA7B;AACD;AACF;AACF,aAdD,MAcO;AACL,qBAAO,OAAKpH,OAAL,CAAakB,OAAb,CAAqB5W,IAArB,CAAP;AACD;AACF;AACF,SA1BD;AA2BD;AACF;;AAED;;;;;;;;0CAK4B;AAAA;;AAC1B,WAAK+X,aAAL,GAAqB,IAAI/R,OAAJ,CAAY,UAACC,OAAD,EAAU6E,MAAV,EAAqB;AACpD,eAAK8K,aAAL,CAAmBoH,UAAnB,SAAoC,2BAAgBpY,cAApD,EAAoE,YAAM;AACxE,iBAAKoS,aAAL,CAAmB,wBAAc,2BAAgBpT,YAA9B,CAAnB;AACAqC;AACD,SAHD;AAIA,eAAK2P,aAAL,CAAmBpP,MAAnB,SAAgC,0BAAelH,KAA/C,EAAsDwL,MAAtD;AACD,OANoB,EAMlByN,KANkB,CAMZ,YAAM;AACb;AACD,OARoB,CAArB;AASD;;AAED;;;;;;;;;8CAMmC;AAAA;;AACjC,UAAM0E,iBAAiB,KAAKvH,OAAL,CAAayC,QAAb,CAAsB8E,cAA7C;AACA,UAAMC,eAAe,KAAKxH,OAAL,CAAayC,QAAb,CAAsB+E,YAA3C;AACA,UAAMrU,UAAU,KAAK6M,OAAL,CAAa7M,OAA7B;AAHiC;AAAA;AAAA;;AAAA;AAAA;AAAA,cAIxBsU,QAJwB;;AAK/B,cAAMC,WAAW,OAAOD,SAASE,MAAhB,KAA2B,QAA3B,GAAsCF,SAASE,MAAT,CAAgBlL,WAAhB,EAAtC,GAAsE,EAAvF;AACA,cAAMmL,SAAS,OAAOH,SAASG,MAAhB,KAA2B,QAA3B,GAAsCH,SAASG,MAAT,CAAgBnL,WAAhB,EAAtC,GAAsE,EAArF;AACA,cAAMqB,SAAS,+BAAeD,UAAf,GAA4BuG,IAA5B,CAAiC;AAAA,mBAAUtG,OAAOhG,EAAP,KAAc4P,QAAxB;AAAA,WAAjC,CAAf;AACA,cAAI5J,MAAJ,EAAY;AACV,gBAAM+J,gBAAgB1U,QAAQyU,MAAR,CAAtB;AACA,gBAAIC,iBAAiBA,cAAczW,MAAd,GAAuB,CAA5C,EAA+C;AAC7C,kBAAMgC,SAASyU,cAAc,CAAd,CAAf;AACA,kBAAI/J,OAAOgK,aAAP,CAAqB1U,MAArB,EAA6BoU,aAAaI,MAAb,CAA7B,EAAmD,OAAK5H,OAAL,CAAa+H,GAAhE,CAAJ,EAA0E;AACxEpK,uBAAOzR,OAAP,CAAe0R,KAAf,CAAqB,mBAArB,EAA0CiK,aAA1C;AACA,uBAAKG,WAAL,CAAiBlK,MAAjB,EAAyB1K,MAAzB;AACA,uBAAKoQ,WAAL,GAAmBkE,QAAnB;AACA,uBAAKhG,WAAL,GAAmBkG,MAAnB;AACA;AAAA,qBAAO;AAAP;AACD;AACF;AACF;AApB8B;;AAIjC,6BAAqBL,cAArB,8HAAqC;AAAA;;AAAA;AAiBpC;AArBgC;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;;AAsBjC,aAAO,KAAP;AACD;;AAED;;;;;;;;;;gCAOYzJ,M,EAAwB1K,M,EAA6B;AAC/D,UAAI,CAAC,KAAKmP,OAAV,EAAmB;AACjB,aAAKA,OAAL,GAAezE,OAAOmK,YAAP,CAAoB7U,MAApB,EAA4B,KAAK4M,OAAjC,EAA0C,KAAKb,SAA/C,CAAf;AACA,aAAK+I,eAAL;AACD,OAHD,MAGO;AACL,YAAI,KAAK3F,OAAL,CAAazK,EAAb,KAAoBgG,OAAOhG,EAA/B,EAAmC;AACjC,eAAKyK,OAAL,CAAa4F,OAAb,CAAqB/U,MAArB,EAA6B,KAAK4M,OAAlC;AACD,SAFD,MAEO;AACL,eAAKuC,OAAL,CAAamB,OAAb;AACA,eAAKnB,OAAL,GAAezE,OAAOmK,YAAP,CAAoB7U,MAApB,EAA4B,KAAK4M,OAAjC,EAA0C,KAAKb,SAA/C,CAAf;AACA,eAAK+I,
```

Comment:
dist folder should not be included - revert this file
check your `.gotignore` or your git tool as this was not supposed to be included at all